README.md

```diff
@@ -183,7 +183,9 @@ If you like this project please give it a star
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
+  
 
+- I would like to thank Cindy Shin for this beautiful design
 - I would like to thank microverse for their amazing school program
```

Comment:
- [Optional] Though you did add a a normal message, kindly include the links to give proper acknowledgment 
- E.g

Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

-  Therefore, please add it to the README of your project.